contracts/OpenSTValue.sol

```diff
@@ -372,6 +399,45 @@ contract OpenSTValue is OpsManaged, Hasher {
         return (amountST, expirationHeight);
     }
 
+    /**
+      *	@notice Verify storage of redemption intent hash
+      *
+      *	@param _uuid UUID for utility token
+      *	@param _redeemer Redeemer address
+      *	@param _redeemerNonce Nonce for redeemer account
+      *	@param _blockHeight Block height at which the Merkle proof was generated
+      *	@param _rlpParentNodes RLP encoded parent nodes for proof verification
+      *
+      *	@return bool status if the storage of intent hash was verified
+      */
+    function verifyRedemptionIntentHashStorage(
```

Comment:
verifyRedemptionIntent as a function name is easy to follow.

Reply:
Changed to `verifyRedemptionIntent` .